kcidb/monitor/subscriptions/create_incidents.py

```diff
@@ -0,0 +1,42 @@
+"""Automate incident creation"""
+import os
+import kcidb
+from kcidb.tools import kcidb_match
+
+CLIENT = None
+
+
+# pylint: disable=global-statement
+def get_client():
+    """Get KCIDB client instance and set it as a global variable"""
+    global CLIENT
+    if not CLIENT:
+        project_id = os.environ.get('GCP_PROJECT')
+        topic_name = os.environ.get('KCIDB_LOAD_QUEUE_TOPIC')
+        if project_id and topic_name:
+            CLIENT = kcidb.Client(project_id=project_id, topic_name=topic_name)
+    return CLIENT
+
+
+def match_test(test):
+    """Generate incident for matching test"""
+    client = get_client()
+    if client:
+        incident_generator = kcidb_match.IncidentGenerator()
+        incidents = incident_generator.generate_incidents_from_test(test)
```

Comment:
Considering that `IncidentGenerator` encapsulates a database client connection, why not create and get it similarly to `get_client()`, instead of creating a new one for *every* object matched?